logstash-exporter.yaml

```
@@ -0,0 +1,47 @@
+package:
+  name: logstash-exporter
+  version: 1.6.1
+  epoch: 0
+  description: Prometheus exporter for Logstash written in Go
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    GITHUB_REPO: "github.com/kuskoman/logstash-exporter"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 387fb36f814e74e81dd14d82a948e61cefa76f5c
+      repository: https://github.com/kuskoman/logstash-exporter
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      packages: ./cmd/exporter/main.go
+      ldflags: -s -w -X ${GITHUB_REPO}/config.Version="${{package.version}}" -X ${GITHUB_REPO}/config.GitCommit="$(git rev-parse --short HEAD)" -X ${GITHUB_REPO}/config.BuildDate="$(date -u +%Y-%m-%dT%H:%M:%SZ)"
+      repository: https://github.com/kuskoman/logstash-exporter
+      output: main
```

Lets give the binary a more suitable name:

```suggestion
      output: logstash-exporter
```